pages/movies.js

```diff
@@ -0,0 +1,103 @@
+import * as React from "react";
+import {
+  Card,
+  CardContent,
+  CardMedia,
+  Typography,
+  Button,
+  CardActionArea,
+  CardActions,
+  Grid,
+  ThemeProvider,
+  createTheme,
+} from "@mui/material";
+import dbConnect from "../lib/dbConnect";
+import Movie from "../models/movie";
+
+// Create a theme for this component
+const moviesTheme = createTheme({
+  palette: {
+    primary: {
+      main: '#66aca6',
+    },
+    secondary: {
+      main: '#233f3d',
+    },
+    background: {
+      paper: "rgba(102, 172, 166, 0.1)",
+    },
+  },
+  typography: {
+    fontFamily: ['"Playfair Display"'],
+  },
+});
+
+export default function MoviesPage({ movies }) {
+
+  return (
+    <ThemeProvider theme={moviesTheme}>
+      <Grid
+        container
+        justifyContent="center"
+        sx={{ maxWidth: 1200, mx: "auto", gap: 2, flexGrow: 1 }}
+      >
+        {movies.map((movie) => (
+          <div key={movie.id}>
```

Comment:
As far as I can tell, this `div` is redundant. You can set the `key` property on the `Grid` item directly, and I believe that should work just as well, but leave the DOM slightly simpler which is usually a good thing.